presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -328,6 +337,12 @@ Property Name                                        Description
 
 ``hive.metastore.glue.external-id``                  External ID for the IAM role trust policy when connecting
                                                      to the Glue Catalog.
+
+``hive.metastore.glue.partitions-segments``          Number of segments for partitioned Glue tables, defaults
+                                                     to ``5``.
+
+``hive.metastore.glue.get-partition-threads``        Number of threads for parallel partition fetches from Glue,
```

Comment:
```suggestion
``hive.metastore.glue.get-partition-threads``        Number of threads for parallel partition fetches from AWS Glue,
```

Reply:
we used Glue without the AWS prefix all over the place. I kinda think we don't need to add it here or in most other places.